webui/react/src/shared/utils/error.ts

```diff
@@ -80,24 +80,23 @@ export class DetError extends Error implements DetErrorOptions {
   /** the wrapped error if one was provided. */
   sourceErr: unknown;
 
-  constructor(e?: unknown, options: DetErrorOptions = {}) {
+  /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+  constructor(e?: any, options: DetErrorOptions = {}) {
     const defaultMessage = isError(e) ? e.message : isString(e) ? e : DEFAULT_ERROR_MESSAGE;
     const message = options.publicSubject || options.publicMessage || defaultMessage;
     super(message);
 
-    const eOpts: DetErrorOptions = isDetError(e)
-      ? {
-          id: e.id,
-          isUserTriggered: e.isUserTriggered,
-          level: e.level,
-          logger: e.logger,
-          payload: e.payload,
-          publicMessage: e.publicMessage,
-          publicSubject: e.publicSubject,
-          silent: e.silent,
-          type: e.type,
-        }
-      : {};
```

Comment:
this was making it so that when you did `handleError` with an object literal like so:

```
handleError({
   publicMessage: ...
   ...
})
```

everything was ignored-- since an object literal is`!isDetError`